src/main/java/com/github/swrirobotics/bags/persistence/Bag.java

```diff
@@ -325,7 +325,7 @@ private void setLongitudeDeg(Double longitudeDeg) {
         return tags;
     }
 
-    private void setTags(Set<Tag> tags) {
+    public void setTags(Set<Tag> tags) {
```

Comment:
For what it's worth, the setter for Collection objects should never be used on Hibernate classes, so it's best to leave them `private` to prevent accidental usage.  Keep in mind that this method is really an abstraction for the relationship between database tables, so setting it to an instance of a different `Set` object would be the equivalent of leaving all of the old `Tag`s in the database unassociated with this `Bag`.  Instead, if you need to modify the tags associated with a bag, it's better to call `getTags` and modify the set returned by that.